rest_framework_jwt/views.py

```diff
@@ -99,6 +101,23 @@ class RefreshJSONWebToken(JSONWebTokenAPIView):
     serializer_class = RefreshJSONWebTokenSerializer
 
 
+class DeviceViewSet(mixins.ListModelMixin, mixins.DestroyModelMixin, viewsets.GenericViewSet):
+    queryset = Device.objects.all()
+    serializer_class = DeviceSerializer
+    permission_classes = [IsAuthenticated]
+
+    def get_queryset(self):
+        return self.queryset.filter(user=self.request.user)
+
+    def get(self, *args, **kwargs):
```

Comment:
We used mixins so router should discover it. Check: http://www.django-rest-framework.org/api-guide/viewsets/